fuels-abigen-macro/tests/harness.rs

```diff
@@ -1323,14 +1327,14 @@ async fn test_amount_and_asset_forwarding() {
         .unwrap();
     assert_eq!(balance_result.value, 0);
 
-    instance.mint_coins(11).call().await.unwrap();
+    instance.mint_coins(5_000_000).call().await.unwrap();
 
     balance_result = instance.get_balance(target, asset_id).call().await.unwrap();
-    assert_eq!(balance_result.value, 11);
+    assert_eq!(balance_result.value, 5_000_000);
 
     // Forward 1 coin of native asset_id
     let tx_params = TxParameters::new(None, Some(1_000_000), None);
-    let call_params = CallParameters::new(Some(1), None);
+    let call_params = CallParameters::new(Some(1_000_000), None);
```

Comment:
We should probably document the fact that using "big" numbers is what is also being tested here

Reply:
What is the best place for documenting it?

Reply:
In a comment in the test